src/ucs/async/thread.c

```diff
@@ -312,7 +311,7 @@ static ucs_status_t ucs_async_thread_add_event_fd(ucs_async_context_t *async,
 
     /* Store file descriptor into void * storage without memory allocation. */
     status = ucs_event_set_add(thread->event_set, event_fd,
-                               UCS_EVENT_SET_EVREAD,
+                               events,
                                (void *)(uintptr_t)event_fd);
```

Comment:
pls, move it on 315th line:
`events, (void *)(uintptr_t)event_fd);`

Reply:
Fixed. Could you please comment about my above concern?

Reply:
Thanks. Moved to `event_fd` place.